crowbar_framework/app/controllers/api/ceph_controller.rb

```diff
@@ -14,23 +14,22 @@
 # limitations under the License.
 #
 
-class Api::StoragesController < ApiController
+class Api::CephController < ApiController
   before_action :set_ceph
 
-  api :GET, "/api/storages", "List all Ceph storages"
+  api :GET, "/api/ceph", "Show the Ceph object"
```

Comment:
I just saw that we don't store the ceph version anywhere as far as i could see. but lets just drop this endpoint for now as we don't really have a use case